demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -119,7 +119,7 @@
 /**
  * @brief Transport timeout in milliseconds for transport send and receive.
  */
-#define TRANSPORT_SEND_RECV_TIMEOUT_MS           ( 500 )
+#define TRANSPORT_SEND_RECV_TIMEOUT_MS           ( 500u )
```

Comment:
Can you make this an uppercase U for consistency?

Reply:
Fixed